frontend/src/components/Election/VoterAuth.tsx

```diff
@@ -67,7 +67,7 @@ const VoterAuth = ({ authSession, electionData, fetchElection }: Props) => {
               </Typography>
             </Box>
           }
-          {voterIdRequired &&
```

Comment:
This disables the voter id box if it was provided via a url parameter

I'm curious to hear feedback on this idea. I'm suggesting it because people generally seem confused to see a prefilled box when they open the ballot, but I could also see an argument that having more context is better

I'm also not sure if the voter needs to be aware of the voter id at all. The election admins should be aware, and voters should have a way to learn about it, but I don't think it needs to be shown during the voting process

Reply:
This is a tough one. I think our goals should be to communicate clearly that they've been given a unique link and are authorized to vote. Currently we don't do that but hiding the box doesn't fix the issue and might make people think the opposite. We should definitely add a line in the email that this is a unique link and not to share it.

What I've seen on other forms is a brief empty form, then a filled-out form, some confirmation, then transition to the page you're trying to go to. Maybe we could put in a loading animation and then a green check or something. 

The clear button can probably be removed if we transition away from storing the voter_id in a cookie. 

You should also add a "&& isAuthorized" to this check. I think if an invalid voter_id is submitted that would be false and we'd want to render the box and error. We should also consider refactoring the voter_auth data sent from the server. It wasn't well thought out so it's a bit janky. 

For near term testing this is ok but we'll want to revisit it.

Reply:
Hmm that is tricky

For now I'll add the && isAuthorized and adjust the language on the email to clarify that it's unique

 Also, in the future I think we should move the voter id from the url. Sensitive data in the url can be intercepted. https://docs.guardrails.io/docs/vulnerability-classes/insecure-access-control/sensitive-data-in-urls
 
Maybe we can try using post using this method https://stackoverflow.com/questions/3915917/make-a-link-use-post-instead-of-get

Reply:
Here's another idea, for when the parameter is specified?

![image](https://github.com/Equal-Vote/star-server/assets/9289903/05f98d4b-98be-4596-944d-c8bf3fa7e98f)